node/Cargo.toml

```diff
@@ -72,6 +72,22 @@ fp-storage = { workspace = true, features = ["std"], optional = true }
 golden-gate-runtime-mainnet = { workspace = true, features = ["std"], optional = true }
 golden-gate-runtime-testnet = { workspace = true, features = ["std"], optional = true }
 
+[dev-dependencies]
+assert_cmd = "2.0.2"
```

Comment:
I prefer to keep all dependencies in the workspace file, even dev-dependencies. But up to you

Reply:
We can do this convention, but let's keep crates for testing as `dev-dependencies` rather than workspace dependencies.